standardtracer/standardcontext.go

```diff
@@ -0,0 +1,48 @@
+package standardtracer
+
+import "sync"
+
+type StandardContext struct {
```

Comment:
@bensigelman do you think it's even worth having a standard tracer?  We haven't done it in other languages.